extensions/ql-vscode/src/model-editor/model-editor-view.ts

```diff
@@ -319,6 +320,18 @@ export class ModelEditorView extends AbstractWebview<
         );
         break;
       }
+      case "telemetry":
+        telemetryListener?.sendUIInteraction(msg.action);
+        break;
+      case "unhandledError":
+        void showAndLogExceptionWithTelemetry(
+          extLogger,
+          telemetryListener,
+          redactableError(
+            msg.error,
+          )`Unhandled error in result comparison view: ${msg.error.message}`,
```

Comment:
```suggestion
          )`Unhandled error in model editor view: ${msg.error.message}`,
```

Reply:
Oops. Thanks for spotting this. I copy/pasted too eagerly.